addons/sale/static/src/js/product_catalog/kanban_record.js

```diff
@@ -62,7 +62,14 @@ export class ProductCatalogKanbanRecord extends KanbanRecord {
         if (this.productCatalogData.readOnly) {
             return;
         }
-        this.productCatalogData.quantity = quantity;
+        if (this.productCatalogData.quantity === quantity) {
+            // This condition is triggered on the following specific condition that the product was already at the minimum quantity possible,
+            // as stated in the sale_stock module then the user inputs a quantity lower than this limit, then we need the record to forcefully update
+            this.props.record.load();
+            this.props.record.model.notify();
+        } else {
+            this.productCatalogData.quantity = quantity;
+        }
```

Comment:
I didn't find a better way to handle that specific case. However, I believe this concerns only sale_stock and shouldn't be in sale. Please mode this logic in your override, and don't call super if it's not needed.